MANIFEST.in

```diff
@@ -9,3 +9,4 @@ recursive-include docs *.rst conf.py Makefile make.bat *.jpg *.png *.gif
 recursive-include shiny/www *
 recursive-include shiny/experimental/www *
 recursive-include shiny/examples *
+recursive-include shiny/ui/dataframe/js/dist *
```

Comment:
What do you think about not including the .map file? It's 500KB, and 100KB gzipped.

This doesn't matter much for regular shiny usage, but all the map files do add up and can be a nontrivial size for shinylive. I'd be happy to address this as a separate issue later though.

Reply:
Sure, that's fine. Probably better to rebuild the JS without sourcemaps than to build it with an external sourcemap and not include it?